packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -151,7 +156,14 @@ export default function (
               return throwMsg === true || err.message.indexOf(throwMsg) >= 0;
             });
```

Comment:
Should we try to handle the promise case here too, to allow async tests to reject.